Libplanet/Blocks/Block.cs

```diff
@@ -101,8 +100,13 @@ public class Block<T> : ISerializable
                 .ToImmutableArray();
         }
 
-        protected Block(SerializationInfo info, StreamingContext context)
-            : this(new RawBlock(info, context))
+        /// <summary>
+        /// Creates a <see cref="Block{T}"/> instance from its serialization.
+        /// </summary>
+        /// <param name="bytes">The serialization of block instance.
+        /// </param>
+        public Block(byte[] bytes)
```

Comment:
As encapsulation-wise `IValue` is a more semantic representation than `byte[]`, IMHO it's unnatural that a function for `IValue` → `Block<T>` is a static method whereas a function for `byte[]` → `Block<T>` is a constructor.  Aren't they inside out?  To me it would be more natural if there are `Block<T>(IValue)` constructor and `Block<T>.Deserialize(byte[])` static method.